examples/seismic/elastic/wavesolver.py

```diff
@@ -53,22 +54,16 @@ def forward(self, src=None, rec1=None, rec2=None, vp=None, vs=None, rho=None,
             Geometry object that contains the source (src : SparseTimeFunction) and
```

Comment:
Maybe src, recn etc...have to be in parameters as well..

Reply:
exactly. Looks like the docstring is broken. This needs fixing